agent/js/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  ignorePatterns: ["dist"],
+  extends: "standard-with-typescript",
+  overrides: [
+    {
+      env: {
+        node: true,
+      },
+      files: [".eslintrc.{js,cjs}"],
+      parserOptions: {
+        sourceType: "script",
+      },
+    },
+  ],
+  parserOptions: {
+    ecmaVersion: "latest",
+    sourceType: "module",
+    project: "tsconfig.json",
+  },
+  rules: {
+    quotes: [2, "single", { avoidEscape: true }],
```

Comment:
can you change it to single?
```suggestion
    quotes: [1, "single", { avoidEscape: true }],
```

Reply:
I actually went to double instead of single.